include/zephyr/bluetooth/iso.h

```diff
@@ -335,19 +335,33 @@ struct bt_iso_cig_param {
 	 */
 	uint8_t num_cis;
 
-	/** @brief Channel interval in us.
+	/** @brief Channel interval in us for SDUs sent from Central to Peripheral.
 	 *
 	 *  Value range BT_ISO_SDU_INTERVAL_MIN - BT_ISO_SDU_INTERVAL_MAX.
 	 */
-	uint32_t interval;
+	uint32_t c_to_p_interval;
 
-	/** @brief Channel Latency in ms.
+	/** @brief Channel interval in us for SDUs sent from Peripheral to Central.
+	 *
+	 *  Value range BT_ISO_SDU_INTERVAL_MIN - BT_ISO_SDU_INTERVAL_MAX.
+	 */
+	uint32_t p_to_c_interval;
+
+	/** @brief Channel Latency in ms for SDUs sent from Central to Peripheral
 	 *
 	 *  Value range BT_ISO_LATENCY_MIN - BT_ISO_LATENCY_MAX.
 	 *
 	 *  This value is ignored if any advanced ISO parameters are set.
 	 */
-	uint16_t latency;
+	uint16_t c_to_p_latency;
+
+	/** @brief Channel Latency in ms for SDUs sent from Peripheral to Central
+	 *
+	 *  Value range BT_ISO_LATENCY_MIN - BT_ISO_LATENCY_MAX.
+	 *
+	 *  This value is ignored if any advanced ISO parameters are set.
+	 */
+	uint16_t p_to_c_latency;
```

Comment:
Since you've added latency to the same commit as interval, you'll either need to split that, or update the commit message and title

Reply:
commit message updated

Reply:
updated the commit message